api/lib/ZwaveClient.ts

```diff
@@ -6029,6 +6030,15 @@ class ZwaveClient extends TypedEventEmitter<ZwaveClientEventCallbacks> {
 
 		node.deviceId = this._getDeviceID(node)
 		node.hasDeviceConfigChanged = zwaveNode.hasDeviceConfigChanged()
+
+		if(node.isControllerNode) {
+			node.rfRegions = this.driver.controller.getSupportedRFRegions()?.toSorted().map(
```

Comment:
Z-Wave JS sorts by region number already. I think this sorts by the string representation of the numeric value?

Reply:
@AlCalzone LOL nope, I wanted to do that but I have to do the sort later. Let me fix it